.env.sample

```diff
@@ -0,0 +1 @@
+WEATHER_API_KEY = "844066ec80254d2bada71850240207"
```

Comment:
This file should be just an example of how to fill it. You shouldn't push your secure data